### DISCUSSION TITLE
Welcome to solo-spice-ql Discussions!

### DISCUSSION THREAD

**Discussion by martinwiesmann** [ITA-Solar/solo-spice-ql#21] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by steinhh** [comment]
Stein Haugan here!

I'm part of the UiO/ITA group which provides the SPICE quick-look software and the SPICE data pipeline (both the software itself and the operation of it). Terje Fredvik (@tfredvik) is in charge of the pipeline and Martin is in charge of the quick-look software (though Peter Young is officially in charge of the "SPICE Quick-look and Data Analysis Working Group"). I'm in charge of the "miscellaneous/advice/assistance"  task for both the quick-look software and the pipeline :).
